ExampleApp/Shared/PaymentSettingTableViewController.swift

```diff
@@ -327,6 +327,8 @@ extension PaymentSettingTableViewController {
             return .fpx
         case rabbitLinepayCell:
             return .rabbitLinepay
+        case OCBCPAOPaymentCell:
+            return .mobileBankingOCBCPAO
```

Comment:
Isn't this a duplicate of line 310? 🤔

Reply:
Fixed [0a6c018](https://github.com/omise/omise-ios/pull/164/commits/0a6c0187af57db1cc0116b750909323e07e6cccb)